src/frontend/src/lib/components/core/Loader.svelte

```diff
@@ -61,6 +68,14 @@
 
 	let progressModal = false;
 
+	const debounceLadBtcAddressTestnet = debounce(loadBtcAddressTestnet);
```

Comment:
Not sure you have seen my last comment given that you ask me for review again, so repeating here: is that a typo?

`debounceLadBtcAddressTestnet`, `...Lad...`?

Reply:
Yes, that's a typo, thanks!